lib/cartopy/_crs.pyx

```diff
@@ -115,7 +116,7 @@ class Globe(object):
 
     def to_proj4_params(self):
         """
-        Create an OrderedDict of key value pairs which represents this globe
+        Creates an OrderedDict of key value pairs which represents this globe
```

Comment:
Look, there's been a lot of finnicky comments from @QuLogic and @pelson on this PR, but the one thing that I feel should be a priority is consistency in the documentation.  I changed this because some docstrings start with (for example) 'creates' and some with 'create'.  I don't particularly care which they are changed to, but they should be the same tense.